.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -98,7 +98,31 @@ public function test_editor_assets_scripts() {
 		$this->go_to( '/wp-admin/post-new.php' );
 		$this->coblocks_block_assets->editor_assets();
 
-		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered,  ) );
+
+		$path = 'wp-content/plugins/coblocks/dist/coblocks-*.js';
+		$files = glob( $path );
+
+		foreach($files as $file) {
+			$dist_script = end(
+				explode('/',
+					explode('.',
+						$file
+					)[0]
+				)
+			);
```

Comment:
This could be written as a one liner to help clarify what it's intentions are.
```suggestion
			$dist_script = pathinfo( $file, PATHINFO_FILENAME );
```